go-controller/pkg/util/multi_network.go

```diff
@@ -1004,6 +1004,10 @@ func IsNetworkSegmentationSupportEnabled() bool {
 	return config.OVNKubernetesFeature.EnableMultiNetwork && config.OVNKubernetesFeature.EnableNetworkSegmentation
 }
 
+func IsRouteAdvertisementsEnabled() bool {
+	return config.OVNKubernetesFeature.EnableMultiNetwork && config.OVNKubernetesFeature.EnableRouteAdvertisements
```

Comment:
For now I am requiring `config.OVNKubernetesFeature.EnableMultiNetwork` to be set because we will rely on a special NAD for the default network and on the NAD controller to process it. It is really explained better and used on a later PR but of course after months of rebases it gets to a point that I need to consolidate changes to minimize the pain.
We could make an effort to not require it but not sure it is worth it.

Reply:
Makes sense; would you add this reasoning, with a short snippet of the required customization of the default network to the PR's description ? Preferably w/ an example of the NAD.